articles/150_ros_command_line.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: ROS Command Line
+permalink: articles/ros_command_line.html
+abstract:
+  This article describes ROS 2 nodes command line capabilities and its syntax.
+author: '[Michel Hidalgo](https://github.com/hidmic)'
+published: true
+---
+
+- This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Overview
+
+As it was the case in ROS 1, ROS 2 nodes allow configuration via command line arguments to a certain degree.
+However, a larger set of configuration options, an ambiguity in remapping rules and parameter assignment syntax (as a result of the leading underscore name convention for hidden resources), a one-to-many relationship between executables and nodes, among others, add up complexity to an otherwise simple interface.
+
+Increasingly precise addressing mechanisms, as well as leading double underscores (`__`) in positional arguments, solve part of the problem.
+In ROS 2, however, command line options are introduced too as these are:
```

Comment:
For me "command line options" includes the "positional" `from:=to` style arguments. Something like `--opt` or `-o` is what I'd call "flags". I'm not sure that's the right wording, but as it's written it's a confusing. Maybe something like:

> In ROS 2, however, additional command line options have been introduced in order to be:

Reply:
Yeah, it's poorly written. I almost rewrote the entire section in beaf7bec3dfc45db907f8c89e30801dcfdc3eba1.